test/fixtures/mui/output.js

```diff
@@ -0,0 +1,14 @@
+import { styled as matStyled } from '@mui/material/styles';
+import sc from '@mui/material/styles';
+const Test1 = matStyled('div').withConfig({
```

Comment:
Do you really want it to transform `matStyled`? According to this issue this isn't wanted: https://github.com/styled-components/babel-plugin-styled-components/issues/224

Reply:
We didn't want to do this in v4, because the `styled` there was not using `styled-components`. For v5, when the `@mui/system` is configured to use `styled-components`, we want the plugin to process these imports.